frontend/views/containers/proposals/ProposalTemplate.vue

```diff
@@ -82,11 +82,15 @@
     template(slot='footer' v-if='!isConfirmation')
       .c-footer
         i.icon-vote-yea
-        span(v-if='groupShouldPropose' v-safe-html='footerVotingExplanation')
+        span(v-if='shouldPropose' v-safe-html='footerVotingExplanation')
         i18n(
-          v-else
+          v-else-if='!groupShouldPropose'
           :args='LTags("strong")'
         ) Your group has less than 3 members, so {strong_}this change will be immediate{_strong} (no voting required).
+        i18n(
+          v-else
+          :args='LTags("strong")'
+        ) The first distribution period is not started yet, so {strong_}this change will be immediate{_strong} (no voting required).
```

Comment:
@Silver-IT it looks like this last `i18n` block here is only used by `DistributionDate.vue`. Then I feel that instead of specifying it here (which is a template component shared by multiple other proposal modals), it would be a better pattern to pass it as `template(#footer='')` slot from `DistributionDate.vue`.

Reply:
Good idea.